docs/changelog.md

```diff
@@ -3,10 +3,35 @@ title: Changelog
 ---
 
 ### Develop
-* [BUGFIX] Restore support for V2 API style custom expectation rendering (#3179)
+
+
+### 0.13.27
+* [FEATURE] Accept row_condition (with condition_parser) and ignore_row_if parameters for expect_multicolumn_sum_to_equal (#3193)
+* [FEATURE] ConfiguredAssetDataConnector for Azure Blob Storage (#3141)
+* [FEATURE] Replace MetricFunctionTypes.IDENTITY domain type with convenience method get_domain_records() for SparkDFExecutionEngine (#3226)
+* [FEATURE] Replace MetricFunctionTypes.IDENTITY domain type with convenience method get_domain_records() for SqlAlchemyExecutionEngine (#3215)
+* [FEATURE] Replace MetricFunctionTypes.IDENTITY domain type with convenience method get_full_access_compute_domain() for PandasExecutionEngine (#3210)
+* [FEATURE] Set foundation for Azure-related DataConnectors (#3188)
+* [FEATURE] Update ExpectCompoundColumnsToBeUnique for V3 API (#3161)
+* [BUGFIX] Fix incorrect schema validation for Azure data connectors (#3200)
+* [BUGFIX] Fix incorrect usage of "all()" in the comparison of validation results when executing an Expectation (#3178)
+* [BUGFIX] Fixes an error with expect_column_values_to_be_dateutil_parseable (#3190)
+* [BUGFIX] Improve parsing of .ge_store_backend_id (#2952)
 * [BUGFIX] Remove fixture parameterization for Cloud DBs (Snowflake and BigQuery) (#3182)
-* [BUGFIX] Fix an error in how ExpectationValidationResults (EVR)s were compared. The bug could cause some EVRs toappear identical even when some result properties differed; the issue primarily affected a small number of internal tests, since EVR comparison is not used as part of the data validation flow. (#3178)
```

Comment:
Why is this bug fix being removed from the changelog?

Reply:
PR 3178 is still in the changelog, just with the one-line Github PR message rather than the manual note that had been left in the Develop header.

Reply:
Because people didn't stop doing PR's..